### DISCUSSION TITLE
SuiteSparse discussions page is now active

### DISCUSSION THREAD

**Discussion by DrTimothyAldenDavis** [DrTimothyAldenDavis/SuiteSparse#420] [Announcements]
## SuiteSparse Discussions

I've enabled the Discussions feature on github for the SuiteSparse repo.

**Comment by hhhhhhharui** [comment]
I encountered an error while compiling SuiteSparse using CMake, stating that the "cholmod.dll" file cannot be found.The error occurs in the "E:/googledownload/SuiteSparse/build/CHOLMOD/Release/" directory.I’ve tried running cmake --install, but the file still can’t be located showing that cannot find "cholmod.dll" file.Could you suggest any solutions or steps I might be missing to resolve this issue?
